Exploration HCM tweaks

juxjux9930 opened the pull request:
🆑 
tweak: The heavy exploration hardsuit is now pathfinder locked instead of all-access.
tweak: The heavy exploration hardsuit now has insulated gloves for all your scavenging needs.
tweak: The heavy exploration hardsuit can now actually fit exploration related items in it's suit storage.
tweak: The heavy exploration hardsuit now uses the proper OBJ sprite.
/ 🆑 

Never made sense to me why the hardsuit was all access instead of being pathfinder locked - There are five voidsuits for the five explorers, a voidsuit for the singular pilot, and the hardsuit, with one pathfinder. Since the hardsuit is situated away from the normal exploration voidsuits, with the pilot voidsuit placed between the two, it's marked as special. And with the five suits for five machete grunts, and then one special suit left alone, with one special machete LEADER, it seems pretty implied what the suit is for. What happens, however, to latejoining pathies, is there's a good chance someone has nicked your suit because it isn't access locked.

Insulated glove change is because it's a special heavy-duty suit and it makes sense that it'd be more resistant to getting zapped. It also makes it more valuable for antags to steal and more useful for away sites that may require door hacking or wire cutting.

Spookerton commented on maps/torch/items/rigs.dm:
```diff
@@ -300,43 +300,61 @@
-	brightness_on = 0.8
 
-/obj/item/weapon/rig/exploration/equipped
+/obj/item/weapon/rig/command/exploration/equipped
```
```suggestion
/obj/item/weapon/rig/exploration/equipped
```

Spookerton commented on maps/torch/torch1_deck5.dmm:
```diff
@@ -4927,7 +4928,7 @@
 	dir = 4
 	},
-/obj/item/weapon/rig/exploration/equipped,
+/obj/item/weapon/rig/command/exploration/equipped,
```
```suggestion
/obj/item/weapon/rig/exploration/equipped,
```

Spookerton commented on maps/torch/items/rigs.dm:
```diff
@@ -300,43 +300,61 @@
+
+/obj/item/clothing/suit/space/rig/command/exploration
+
+/obj/item/clothing/shoes/magboots/rig/command/exploration
```
```suggestion
```

Spookerton commented on maps/torch/items/rigs.dm:
```diff
@@ -300,43 +300,61 @@
 
+	req_access = list(access_pathfinder)
+
+/obj/item/clothing/suit/space/rig/command/exploration
```
```suggestion
```

Spookerton commented on maps/torch/items/rigs.dm:
```diff
@@ -300,43 +300,61 @@
-				   /obj/item/device/flashlight,
-				   /obj/item/weapon/tank,
-				   /obj/item/device/suit_cooling_unit)
+	boot_type = /obj/item/clothing/shoes/magboots/rig/command/exploration
```
```suggestion
	boot_type = /obj/item/clothing/shoes/magboots/rig/command
```

Spookerton commented on maps/torch/items/rigs.dm:
```diff
@@ -300,43 +300,61 @@
 		)
 
-	chest_type = /obj/item/clothing/suit/space/rig/command
+	chest_type = /obj/item/clothing/suit/space/rig/command/exploration
```
```suggestion
	chest_type = /obj/item/clothing/suit/space/rig/command
```

Spookerton commented on maps/torch/items/rigs.dm:
```diff
@@ -300,43 +300,61 @@
 * EXPLORATION
 */
-/obj/item/weapon/rig/exploration
+/obj/item/weapon/rig/command/exploration
```
```suggestion
/obj/item/weapon/rig/exploration
```

Spookerton reviewed (dismissed):
paths are inheritance, not concept hints

juxjux9930 committed:
Exploration HCM tweaks


Update maps/torch/items/rigs.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update maps/torch/torch1_deck5.dmm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update maps/torch/items/rigs.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update maps/torch/items/rigs.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update maps/torch/items/rigs.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update maps/torch/items/rigs.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update maps/torch/items/rigs.dm

Co-authored-by: Spookerton <spkrtn@pm.me>

juxtaposed9930 commented:
Undid my sins on request, Spook.

Spookerton commented on maps/torch/items/rigs.dm:
```diff
@@ -306,35 +306,51 @@
 	camera = /obj/machinery/camera/network/exploration
 	icon_state = "command_exp_rig"
 	light_overlay = "yellow_double_light"
-	brightness_on = 0.8
```
This makes the helmet light dimmer, the same as non-exploration helmets. Intended?

juxjux9930 replied:
Dimmer? Huh, was under the impression the default was brighter, but I just checked and you're right. No, not intended, and I'll fix it in a bit.

comma commented:
> Never made sense to me why the hardsuit was all access instead of being pathfinder locked
ftr it was because there's nothing inherent 'pathfinder' to that suit, it's just a heavy suit. it doesn't do anything better for team leader duties,
it's not even that great, and if team is full, pathfinder is forced to use it because no one else can
and lastly 'command hardsuits' are disgusting concept that reinforces 'head is super[job]' and i wanted to at least fix it in explo team
